src/components/Frame/context.tsx

```diff
@@ -11,15 +10,6 @@ export interface FrameContextType {
   stopLoading(): void;
 }
 
-const defaultContext: FrameContextType = {
-  showToast: noop,
-  hideToast: noop,
-  setContextualSaveBar: noop,
-  removeContextualSaveBar: noop,
-  startLoading: noop,
-  stopLoading: noop,
-};
-
-const FrameContext = React.createContext<FrameContextType>(defaultContext);
+const FrameContext = React.createContext<FrameContextType | null>(null);
```

Comment:
`Sheet` checks `frame == null` but since we always have defaults with the new context API it'll always be true. So I'm defaulting to null and adding more explicit checks below